frontend/src/api/Game.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { axiosErrorHandler } from './Error';
+import { User } from './User';
+
+export type StartGameParams = {
+  initiator: User,
+};
+
+export const startGame = (params: StartGameParams):
+  Promise<void> => axios.post<void>('/api/v1/rooms/{currentRoomId}/start', params)
```

Comment:
Change to the following (+ add import for Room): 

```suggestion
export const startGame = (roomId: string, params: StartGameParams):
  Promise<Room> => axios.post<Room>(`/api/v1/rooms/${roomId}/start`, params)
```